ckks/bootstrapp.go

```diff
@@ -14,12 +13,27 @@ func (btp *Bootstrapper) Bootstrapp(ct *Ciphertext) *Ciphertext {
 	//var t time.Time
 	var ct0, ct1 *Ciphertext
 
-	for ct.Level() != 0 {
+	for ct.Level() != 1 {
 		btp.evaluator.DropLevel(ct, 1)
 	}
 
 	// Brings the ciphertext scale to Q0/2^{10}
-	btp.evaluator.ScaleUp(ct, math.Round(btp.prescale/ct.Scale()), ct)
+	if ct.Level() == 1 {
```

Comment:
This behavior needs to be documented in the GoDoc above.

Reply:
I updated the godoc